planner/property/physical_property.go

```diff
@@ -51,8 +53,25 @@ const (
 	BroadcastType
 	// HashType requires current task to shuffle its data according to some columns.
 	HashType
+	// PassThroughType requires all the task pass the data to one node (tidb/tiflash).
```

Comment:
PassThrough doesn't make sense. The name should indicate that the tasks should gather into one node.

Reply:
@fzhedu

Reply:
I think MPPPartitionType is different from ExchangeType, MPPPartitionType defines how the data looks upstream and downstream, while ExchangeType defines the transformation. "PassThrough" is not an accurate word to describe data distribution, but a word for action.

For scalability in the future, it's better to separate the 2 definitions clearly. Maybe in the future is not 1-to-1 mapping. Prefer a word like "SinglePartitionType", in case someday PassThrough exchange does not only generate SinglePartitionType. 

For example, if someday we separate computing and storage, 2 nodes can pass through to 1 node for transformation from 200 hash partitions to 100 hash partitions with the same hash keys.